components/MusicPlayer/MusicPlayer.vue

```diff
@@ -89,6 +89,8 @@
             progress = state.progress_ms + (Date.now() - songProgress.value.fetched);
         }
 
+        progress = Math.min(progress, state.item.duration_ms);
```

Comment:
Ist das nicht überflüssig nach der einführung von `overflow: hidden`?

Reply:
nein. progress ist die zeit in ms und wird für die zeit anzeige verwendet